racingcar-java/src/main/java/racingcar5/Car.java

```diff
@@ -0,0 +1,40 @@
+package racingcar5;
+
+public class Car {
+
+    private MoveStrategy moveStrategy;
+    private String name;
+    private Position position;
```

Comment:
```suggestion
    private final MoveStrategy moveStrategy;
    private final String name;
    private final Position position;
```
특별한 트레이드오프의 영역이 아니라면 클래스 변수에 대한 불변은 지켜주세요 :)

Reply:
다시 점검 하겠습니다!
피드백 감사합니다 :)